bootstrap.php

```diff
@@ -12,3 +12,5 @@ function ( $features ) {
 }
 
 new PerformanceFeatureHooks();
+
+require_once __DIR__ . '/includes/BurstSafetyMode/init.php';
```

Comment:
Are we planning to include this in the plugin's alternate experience instead of here?

Reply:
yes, we'll have a code there too but it's needed have that in the module too because when the burst safety mode is disabled the alternate experience folder is not loaded but we need anyway include the init file of burst safety folder to restore cache settings